query/src/sql/planner/binder/bind_context.rs

```diff
@@ -52,6 +53,8 @@ pub struct BindContext {
     /// non-grouping columns cannot be referenced outside aggregation
     /// functions, otherwise a grouping error will be raised.
     pub in_grouping: bool,
+
+    pub explain_kind: Option<ExplainKind>,
```

Comment:
I put `explain_kind` in `BindContext`, do you have any better thoughts? @leiysky

Reply:
It's better to add a plan for `EXPLAIN`.

Reply:
Ok, I'll add it

Reply:
fixed